### PR TITLE
Fix ui

### LINKED ISSUE #74: Style Chat Component
- [ ] Position it and make it looks good 
- [ ] Have it be a scrollable container (there is functionality the make it scroll to the bottom when new messages are received, so make sure it can play nicely with that)

### PR DESCRIPTION
**Pull request by Chriscastanuela**
## Description
* changed `Gamepage` UI and included SCSS throughout the app

## Screenshots (if applicable):
Screen Shot 2020-11-04 at 7 16 57 AM

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Test A - ran in DOM

## Relevant Tickets
Closes #74 

## Tags
@GreyMatteOr 
@holladayian

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^5.11.5",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "node-sass": "4.14.1",
```
Comment (GreyMatteOr) [inline package.json:10]: I knew it was only a matter of time.

**Review by GreyMatteOr — APPROVED** [review]
Looks SO much better.  The gamepage layout will probably have to change just 1 one more time and then it will be done. 

NOTE: 

I really want to add a page that says 'Waiting for other players to Join' after a person clicks to be the `generator`.

File: src/App.scss [hunk 2]
```diff
@@ -1,3 +1,12 @@
+@mixin headerButtons {
```
Comment (holladayian) [inline src/App.scss:1]: Awesome! Thanks for adding mixins! This looks super nice!

**Review by holladayian — APPROVED** [review]

**Comment by holladayian** [comment]
Wow, this was awesome! Thanks for getting all of this done! The only little "nit-picky" comment I have is, do you mind if we put the guess input and the chat input as the inner two sections? Other than that, this baby looks like it's ready to turn in!

**Pull request merged**